### PR TITLE
describe general approach for benchmarking and describe npm-proxy case

### PR DESCRIPTION
**Pull request by brastak**
For #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,4 +7,82 @@
...
+
+Initial setup is mandatory part for every test scenario. Empty repositories could not be
+representative for the tests so we need to fill up them in the beginning. The best way
+for hosted repositories is to import some existing repository for it. The good way for
+proxy repository - to build a number of popular projects to fill it (the best way is
+still import existing data if you have it).
+
+##  Per-repository testing
+
+1. [NPM](npm/README.md)
```
Comment (g4s8) [inline README.md]: @brastak I'd add `files-adapter` too, it's the simplest repository type used to download and upload binary files (like C, C++ libraries).

**Review by g4s8 — CHANGES_REQUESTED** [review]
@brastak just one comment

File: README.md (hunk 1 above)
Comment (brastak) [reply README.md]: I have added stub only, benchmarking details and results will be added in separate issue

**Comment by brastak** [comment]
@g4s8 Please, see update

**Pull request merged**